app/plugins/router.js

```diff
@@ -49,11 +39,26 @@ if (config.endemics.enabled) {
   )
 }
 
+if (config.devLogin.enabled) {
+  routes = routes.concat(require('../routes/endemics/dev-sign-in').handlers)
+}
+
 module.exports = {
   plugin: {
     name: 'router',
     register: (server, _) => {
+      routes.filter(x => !x.path.includes('assets') && !x.path.includes('health'))
+        .forEach(route => assignPreRouteFunction(route, server))
```

Comment:
This feels like a bit of a side effect, especially as the loggingContext is added elsewhere. Might it be clearer to stick all this in hook which attaches a 'preHandler` event similar to [error pages](https://github.com/DEFRA/ffc-ahwr-farmer-claim/blob/main/app/plugins/error-pages.js#L5)?

Probably we don't care if it runs agains health endpoints either but we can always filter them out there if we want to.